src/System.h

```diff
@@ -5,6 +5,8 @@
 
 namespace System
 {
+    std::string GetWorkspaceSymbol(int index);
```

Comment:
Can you put that to the other Workspace related stuff? (i.e. inside of the Hyprland #ifdef)